tests/e2e-ui-tests/utils/constants.js

```diff
@@ -0,0 +1,130 @@
+module.exports = {
```

Comment:
This entire file is no longer needed. It has been replaced by the files in the data folder which splits the data into various sections.